WcaOnRails/spec/features/edit_user_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.feature "Edit user" do
+  let(:admin) { FactoryBot.create(:admin) }
+  let(:existing_user) { FactoryBot.create(:user_with_wca_id) }
+  let(:new_person) { FactoryBot.create(:person) }
+  let(:new_user) { FactoryBot.create(:user) }
+
+  def navigate_to_form(user)
+    visit edit_user_path(user)
+  end
+
+  def submit_form
+    find('input[type="submit"]').click
+  end
+
+  scenario "entering wca id", js: true do
+    sign_in admin
+    navigate_to_form(new_user)
+
+    # Entering an existing wca id
+    fill_in "WCA ID", with: existing_user.wca_id
+    find('input[type="submit"]').click
```

Comment:
This could use the `submit_form` method you created above =)

Reply:
Of course. I was too hurried there. My apologies. :)